foundation/foundation-mda/src/main/java/com/boozallen/aiops/mda/generator/AirflowDockerPomGenerator.java

```diff
@@ -89,13 +89,6 @@ public void generate(GenerationContext context) {
             generateFile(context, vc);
 
             manualActionNotificationService.addNoticeToAddModuleToParentBuild(context, airflowDockerArtifactId, "docker");
-            DockerBuildParams params = new DockerBuildParams.ParamBuilder()
-                    .setContext(context)
-                    .setAppName("airflow")
-                    .setDockerApplicationArtifactId(airflowDockerArtifactId)
-                    .setDockerArtifactId(context.getArtifactId())
-                    .setIncludeLatestTag(true).build();
-            manualActionNotificationService.addDockerBuildTiltFileMessage(params);
```

Comment:
Q: Can we delete the `manualActionNotificationService.addDockerBuildTiltFileMessage(params)` method?

Reply:
this function is still used for the inference/training pipeline. Should be able to remove it altogether on the follow up ticket